test/api/parameterizer.spec.js

```diff
@@ -0,0 +1,99 @@
+var assert = require('assert'),
+    sinon = require('sinon');
+
+var parameterizer = require('../../lib/dictionaries/api/parameterizer');
+
+
+
+describe('Parameterizer', function() {
+
+    var createMockResponse = function() {
```

Comment:
Since all the specs are referring to the default parameteriser I would add a describe block for it and set a context variable with the value of that particular function:

```javascript
var parameterizer = require('../../lib/dictionaries/api/parameterizer')();
describe('Parameterizer', function() {
   describe('#default', function(){
      beforeEach(function(){
        this.subject = parameterizer.default;
      });
     // Every test here, which would use this.subject instead of parameterizer().default
   });
});
```